.ci/aarch64/clean_up_aarch64.sh

```diff
@@ -4,10 +4,8 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-set -e
-
 tests_repo="${tests_repo:-github.com/kata-containers/tests}"
 lib_script="${GOPATH}/src/${tests_repo}/.ci/lib.sh"
 source "${lib_script}"
 
-gen_clean_arch
+gen_clean_arch || true
```

Comment:
- Having both "`set -e` removed" AND "`|| true` added" is probably redundant?
- What about replacing the `||true` with something like:  `|| echo "WARNING: Arch clean up failed"`

Reply:
Yeah, I like that - I'll do the rework.
The patches got the metrics slave running (it now fails as I have to set the check bounds for that machine....)

Reply:
Updated. I used the available `info` func call to dump the info.